gubled/gubled.go

```diff
@@ -108,29 +142,38 @@ var CreateModules = func(router server.Router, args Args) []interface{} {
 func Main() {
 	defer func() {
 		if p := recover(); p != nil {
-			protocol.Err("%v", p)
-			os.Exit(1)
```

Comment:
BECAUSE ..
logger.Fatal("Fatal error in gubled after recover") 
// Calls os.Exit(1) after logging
log.Fatal("Bye.")